v03_pipeline/lib/model/cached_reference_dataset_query.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any, Callable
+
+import hail as hl
+
+from hail_scripts.computed_fields.vep import (
+    CONSEQUENCE_TERM_RANK_LOOKUP,
+    get_expr_for_vep_sorted_transcript_consequences_array,
+    get_expr_for_worst_transcript_consequence_annotations_struct,
+)
+from hail_scripts.reference_data.clinvar import (
+    CLINVAR_PATHOGENICITIES_LOOKUP,
+    parsed_clnsig,
+)
+
+from v03_pipeline.lib.model.definitions import ReferenceGenome
+
+CLINVAR_PATH_RANGE = ('Pathogenic', 'Pathogenic/Likely_risk_allele')
+CLINVAR_LIKELY_PATH_RANGE = ('Pathogenic/Likely_pathogenic', 'Likely_risk_allele')
+GNOMAD_HIGH_AF_THRESHOLD = 0.90
+
+
+def clinvar_path_variants(
+    ht: hl.Table,
+    **_: Any,
+) -> hl.Table:
+    pathogenicity = parsed_clnsig(ht)[0]
+    pathogenicity_id = CLINVAR_PATHOGENICITIES_LOOKUP.get(pathogenicity)
+    ht = ht.select(
+        pathogenic=(
+            CLINVAR_PATHOGENICITIES_LOOKUP.contains(pathogenicity)
+            & (
+                pathogenicity_id
+                >= CLINVAR_PATHOGENICITIES_LOOKUP[CLINVAR_PATH_RANGE[0]]
+            )
+            & (pathogenicity_id < CLINVAR_PATHOGENICITIES_LOOKUP[CLINVAR_PATH_RANGE[1]])
+        ),
+        likely_pathogenic=(
+            CLINVAR_PATHOGENICITIES_LOOKUP.contains(pathogenicity)
+            & (
+                pathogenicity_id
+                >= CLINVAR_PATHOGENICITIES_LOOKUP[CLINVAR_LIKELY_PATH_RANGE[0]]
+            )
+            & (
+                pathogenicity_id
+                < CLINVAR_PATHOGENICITIES_LOOKUP[CLINVAR_LIKELY_PATH_RANGE[1]]
+            )
+        ),
+    )
+    return ht.filter(ht.pathogenic | ht.likely_pathogenic)
+
+
+def gnomad_coding_and_noncoding_variants(
+    ht: hl.Table,
+    reference_genome: ReferenceGenome,
+    **_: Any,
+) -> hl.Table:
```

Comment:
It does not.  I think whatever docs we'd want might live [down here](https://github.com/broadinstitute/seqr-loading-pipelines/pull/556/files#diff-982d56a3e5ca7e0ebb8060542f720a9a332499bf0d937031be9dafe40864e069R121) anyways, since these are little helper functions with a certain type signature.